utils/rpms/daos.spec

```
@@ -360,6 +361,7 @@ mkdir -p %{buildroot}/%{_sysconfdir}/ld.so.conf.d/
 echo "%{_libdir}/daos_srv" > %{buildroot}/%{_sysconfdir}/ld.so.conf.d/daos.conf
 mkdir -p %{buildroot}/%{_sysctldir}
 install -m 644 utils/rpms/%{sysctl_script_name} %{buildroot}/%{_sysctldir}
+install -m 644 utils/rpms/%{limits_script_name} %{buildroot}/%{_sysconfigdir}/security/limits.d
```

I should have noticed this before, but you need a `mkdir` here like you did for debian: https://github.com/daos-stack/daos/blob/fa0f5e8bdfa1ca42b248f0eb65e8b05ca3e0795b/debian/rules#L58

```suggestion
mkdir -p ${buildroot}$(sysconfdir)/security/limits.d
install -m 644 utils/rpms/%{limits_script_name} %{buildroot}/%{_sysconfigdir}/security/limits.d
```